cmd/mesh_bridge/main.go

```diff
@@ -0,0 +1,106 @@
+// +build !js
+
+// mesh_bridge is a short program that bridges two Mesh nodes. This is useful in cases where
+// we introduce a network-level breaking change but still want the liquidity from one network
+//to flow to another
+package main
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/0xProject/0x-mesh/rpc"
+	"github.com/0xProject/0x-mesh/zeroex"
+	"github.com/plaid/go-envvar/envvar"
+	log "github.com/sirupsen/logrus"
+)
+
+type clientEnvVars struct {
+	FirstWSRPCAddress  string `envvar:"FIRST_WS_RPC_ADDRESS"`
+	SecondWSRPCAddress string `envvar:"SECOND_WS_RPC_ADDRESS"`
+	Verbose            bool   `envvar:"VERBOSE"`
+}
+
+func main() {
+	env := clientEnvVars{}
+	if err := envvar.Parse(&env); err != nil {
+		panic(err)
+	}
+
+	if env.Verbose {
+		log.SetLevel(log.Level(4))
+	} else {
+		log.SetLevel(log.Level(0))
+	}
+
+	firstClient, err := rpc.NewClient(env.SecondWSRPCAddress)
+	if err != nil {
+		log.WithError(err).Fatal("could not create client")
+	}
+	stats, err := firstClient.GetStats()
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.WithField("stats", stats).Info("Spun up first client")
+
+	secondClient, err := rpc.NewClient(env.FirstWSRPCAddress)
+	if err != nil {
+		log.WithError(err).Fatal("could not create client")
+	}
+	stats, err = secondClient.GetStats()
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.WithField("stats", stats).Info("Spun up second client")
+
+	wg := &sync.WaitGroup{}
```

Comment:
You've set up this `WaitGroup` so that we can wait for `pipeOrders` to return, but because of the way the function is written, it never does. We're waiting for nothing. We should either:

1.  Make `pipeOrders` return an error if something goes wrong, and then handle that error by e.g. canceling a context. Look at [how we do this in the `core` package](https://github.com/0xProject/0x-mesh/blob/1a4a775b4e6c6f726d004a1e897ffc88bc197180/core/core.go#L470).
2. Remove this `WaitGroup` and just exit the program if something goes wrong in `pipeOrders` with `log.Fatal`.